yarn.lock

```diff
@@ -0,0 +1,11940 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file is intentionally not included with the theme and should not be included in this PR.